ispdb/janis.or.jp.xml

```diff
@@ -1,23 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
   <emailProvider id="janis.or.jp">
-    <!-- ytg.janis.or.jp etc. use MX mx1.janis.or.jp -->
     <domain>janis.or.jp</domain>
     <displayName>Janis</displayName>
     <displayShortName>Janis</displayShortName>
+    <incomingServer type="imap">
+      <hostname>imaps.janis.or.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILLOCALPART%</username>
```

Comment:
issue: the documentation states (in the yellow table, right below SMTP settings, I'm translating through DeepL):

> Username or account name: Email address of the User Information Notice

Which I understand to mean we should use the whole email address. Same comment applies to POP3 and SMTP.